src/lib/draw/SplitRouteMode.svelte

```diff
@@ -92,10 +102,20 @@
           // "Deep clone"
           piece2.properties = JSON.parse(JSON.stringify(piece1.properties));
 
-          fixRouteProperties(gj.features[snappedIndex], piece1, piece2, cursor);
+          fixRouteProperties(
+            gj.features[snappedIndex] as Feature<LineString>,
+            piece1,
+            piece2,
+            cursor
+          );
 
           // Replace the one LineString we snapped to with the two new pieces
-          gj.features.splice(snappedIndex, 1, piece1, piece2);
+          gj.features.splice(
+            snappedIndex,
+            1,
+            piece1 as OurFeature<LineString>,
```

Comment:
We've been dealing with the generic geojson Feature for piece1 and piece2. Above we added in properties, set the ID, etc and made it a valid feature for our definition. So this is another case where static analysis can't figure it out, and we just say "trust us"